bigtable/tests/instance_admin_emulator.cc

```diff
@@ -155,7 +168,13 @@ class InstanceAdminEmulator final
   grpc::Status ListClusters(grpc::ServerContext* context,
                             btadmin::ListClustersRequest const* request,
                             btadmin::ListClustersResponse* response) override {
-    return grpc::Status(grpc::StatusCode::UNIMPLEMENTED, "not implemented");
+    std::string prefix = request->parent() + "/clusters/";
```

Comment:
This works well except in the case where `request->parent()` ends in the magical value `/instances/-`.  In that case you want to return all the clusters in the project.  I think you want to change this to:

```C++
std::string prefix = request->parent();
auto pos = prefix.find("/instances/-"); 
if (std::string::npos == pos) {
  prefix += "/clusters/";
} else {
  prefix = prefix.substr(0, pos); // check me here, I always forget how substr() works.
}
```